notebooks/query-language-object-based-dialect/06.01.predicate_combination_conjunction.ipynb

```diff
@@ -0,0 +1,422 @@
+       "\n",
+       "If a node contains a single metric, the DataFrame will use an `Index` object containing the node column. If a node has an additional level of information, Hatchet creates a `MultiIndex` to store the information pertaining to multiple sets of metrics per node. `MultiIndex` stores the node column as the \"top\" level of the index, followed by additional information on the levels below. \n",
+       "\n",
+       "Based on the types of indexing (`Index or MultiIndex`), retrieving data from a DataFrame corresponding to a particular node either retrieves a single or multiple rows. This difference can cause issues when applying query node predicates.\n",
+       "Therefore, it is necessary to get rid of all index levels besides the node column through an aggregation operation on the GraphFrame. Then, a query node predicate can be applied to the GraphFrame. "
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# why use drop index levels\n",
+    "display(Markdown(\"../common/documentation/drop-index-information.md\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "5478d1cb-5221-49b8-8093-907950fce030",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gf.drop_index_levels()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "37a9c69d-9a25-4ba0-a22e-66a464647fa1",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    },
+    "tags": [
+     "hide-input"
+    ]
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/markdown": [
+       "### Use intersection (AND) in query node predicates\n",
+       "\n",
+       "This query type allows a user to search for nodes that meet the requirements of two or more predicates for a single quantifier. The predicates are combined using the **AND** logical operator means that both predicates must be satisfied by a node.\n",
+       "\n",
+       "The following query matches all nodes where the `time` metric is `greater than 70000` **AND** the `name` metric `starts with MPI_`.\n",
+       "\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Markdown object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# display query type 1 documentation\n",
+    "display(Markdown(\"../common/documentation/predicate-combination-conjunction-01.md\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "83c28c99-c418-442f-bbe6-8ec3a577a817",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "query_1 = [\n",
+    "    (\"*\", {\"time\": \"> 70000\",\n",
```

Comment:
Is **AND** designated/inferred by the comma here? Maybe it should be explained? Perhaps it's obvious but it took me a second.

```suggestion
    "    (\"*\", {\"time\": \"> 70000\",\n", # Comma between dictionary entries is implicit AND
```

Reply:
Yes, AND is achieved with a comma for the Object-based Dialect. You're right, I will add a few words to explain it for this dialect.